src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
```suggestion
    private final Random random = new Random();
    
    public Color getRandomColor() {
        int index = random.nextInt(Color.values().length);
        return Color.values()[index];
```
Random should be a variable class

Reply:
fixed